aws/resource_aws_wafregional_rule.go

```diff
@@ -0,0 +1,193 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/waf"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsWafRegionalRule() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsWafRegionalRuleCreate,
+		Read:   resourceAwsWafRegionalRuleRead,
+		Update: resourceAwsWafRegionalRuleUpdate,
+		Delete: resourceAwsWafRegionalRuleDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"metric_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
It seems the metric name is not actually required per docs:
https://docs.aws.amazon.com/waf/latest/APIReference/API_regional_Rule.html 🤔
Do you mind double checking it and eventually reflecting it here and in the CRUD?

Reply:
Ignore me here - it is indeed required: https://docs.aws.amazon.com/waf/latest/APIReference/API_regional_CreateRule.html